src/convertToObject.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes string with styles
- * and returns object where CSS properties are keys
- * and values are the values of related CSS properties
-
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrStyles = sourceString.split(';').map(style => {
+    return style.trim();
+  }).filter(style => style.length > 0);
```

Comment:
You can write it simpler.
Also, if we use chained methods, each call should be on the same level.
```suggestion
  const arrStyles = sourceString
    .split(';')
    .map(style => style.trim())
    .filter(style => style.length > 0);
```